### PR TITLE
updates the list of users for slack notification

### PR DESCRIPTION
**Pull request by p0lyn0mial**
**What this PR does / why we need it**: updates the list of users for slack notification

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -233,6 +233,11 @@ pipeline:
       - p0lyn0mial=lukasz
       - realfake=luk
       - scheeles=sebastian
+      - kdomanski=kamil
+      - kron4eg=artiom.diomin
+      - thz=thz
```
Comment (p0lyn0mial) [inline .drone.yml]: If the names are the same not sure if we need mapping.

**Review by mrIncompetent — DISMISSED** [review]

File: .drone.yml (hunk 1 above)
Comment (kdomanski) [reply .drone.yml]: explicit > implicit

**Review by metalmatze — CHANGES_REQUESTED** [review]
Can we remove luk please?

File: .drone.yml (hunk 1 above)
Comment (p0lyn0mial) [reply .drone.yml]: @kdomanski not sure if I understand your comment, these are the mappings that maps github users to slack users.

**Comment by p0lyn0mial** [comment]
> Can we remove luk please?

goodbye luk :(

**Review by mrIncompetent — APPROVED** [review]

**Review by metalmatze — APPROVED** [review]

**Comment by metalmatze** [comment]
Thanks!

File: .drone.yml (hunk 1 above)
Comment (kdomanski) [reply .drone.yml]: I meant that it's better to map explicitly even when they're the same, rather than omit the entry from the list.

**Pull request merged**